websites/D/Doordash/dist/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "astrid",
+		"id": "852656702037164053"
+	},
+	"service": "Doordash",
+	"description": {
+		"en": "Delivery & takeout from the best local restaurants. Breakfast, lunch, dinner and more, delivered safely to your door."
+	},
+	"url": "www.doordash.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Y1Yj7ue.png",
+	"thumbnail": "https://i.imgur.com/Sc5sj8U.jpg",
+	"color": "#ff3008",
+	"category": "other",
+	"tags": [
+		"food",
+		"delivery",
+		"restaurant"
+	],
+	"settings": [
+		{
+			"id": "hideFood",
+			"title": "Hidden Food Items",
+			"value": false
+		},
```

Comment:
The title doesn't sound right here. Can you briefly explain what this setting does?

Reply:
It hides the name of the food you're looking at. For example, if you click an item at a restaurant, it shows the item you're looking at within the presence. However, if that setting was enabled, it would just say you're browsing the menu there.

What other title do you think would fit it better?

Reply:
I've changed it to "Hide Food Selection", that should be better